configs/EMAX_F405_ELRS_AIO/config.h

```diff
@@ -0,0 +1,108 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU       STM32F405
+#define BOARD_NAME          EMAX_F405_ELRS_AIO
+#define MANUFACTURER_ID     EMAX 
+
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P 
+#define USE_ACC_SPI_ICM42688P 
+
+#define BEEPER_PIN  PB4
+#define MOTOR1_PIN  PB0
+#define MOTOR2_PIN  PB1
+#define MOTOR3_PIN  PA3
+#define MOTOR4_PIN  PA2
+#define MOTOR5_PIN  PC8
+#define MOTOR6_PIN  PA8
+#define LED_STRIP_PIN   PB6
+#define UART1_TX_PIN    PA9
+#define UART3_TX_PIN    PB10
+#define UART4_TX_PIN    PA0
+#define UART6_TX_PIN    PC6
+#define UART1_RX_PIN    PA10
+#define UART3_RX_PIN    PB11
+#define UART4_RX_PIN    PA1
+#define UART5_RX_PIN    PD2
+#define UART6_RX_PIN    PC7
+#define INVERTER3_PIN   PC9
+#define INVERTER6_PIN   PC8
+#define LED1_PIN        PB5
+#define SPI1_SCK_PIN    PA5
+#define SPI2_SCK_PIN    PB13
+#define SPI3_SCK_PIN    PC10
+#define SPI1_SDI_PIN    PA6
+#define SPI2_SDI_PIN    PB14
+#define SPI3_SDI_PIN    PC11
+#define SPI1_SDO_PIN    PA7
+#define SPI2_SDO_PIN    PB15
+#define SPI3_SDO_PIN    C12
+#define BARO_CS_PIN     PB3
+#define ESCSERIAL1_PIN  PB8
+#define ADC_BATT_PIN    PC2
+#define ADC_RSSI_PIN    PC0
+#define ADC_CURR_PIN    PC1
+#define FLASH_CS_PIN    PB12
+#define MAX7456_CS_PIN  PA15
+#define GYRO_1_EXTI_PIN PC4
+#define GYRO_1_CS_PIN   PA4
+#define USB_DETECT_PIN  PC5
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP(0,PB8 , 2, -1) \
+    TIMER_PIN_MAP(1,PC8 , 2, -1) \
+    TIMER_PIN_MAP(2,PB0 , 2, 0) \
+    TIMER_PIN_MAP(3,PB1 , 2, 0) \
+    TIMER_PIN_MAP(4,PA3 , 1, 1) \
+    TIMER_PIN_MAP(5,PA2 , 1, 0) \
+    TIMER_PIN_MAP(6,PB6 , 1, 0) \
+    TIMER_PIN_MAP(7,PA8 , 1, -1) \
+    TIMER_PIN_MAP(8,PA9 , 1, 0) \

```

Comment:
Why do these pins have timer assignments?

Reply:
if this is resolved, please click `Resolve conversation` button

Reply:
Resolved